content/tutorials/webrtc/datachannels/en/index.html

```diff
@@ -180,46 +179,46 @@ <h2 id="just-show-me-the-action">Configuring data channels</h2>
+  <li><code>negotiated</code>: if set to true, removes the automatic setting up of a data channel on the other peer, providing your own way to create a data channel with the same ID on the other side</li>
+  <li><code>id</code>: allows you to provide your own ID for the channel that can only be used in combination with <code>negotiated</code> set to <code>true</code>)</li>
 </ul>
 
-<p>The only options most people will need to use are the first three: <code>ordered</code>, <code>maxPacketLifeTime</code>, and <code>maxRetransmits</code>. With <a href="https://en.wikipedia.org/wiki/Stream_Control_Transmission_Protocol" title="Wikipedia article about SCTP">SCTP</a> (now used by all browsers that support WebRTC) reliable and ordered is true by default. It makes sense to use unreliable and unordered if you want full control from the application layer, but in most cases, partial reliability is helpful.</p>
+<p>The only options that most people need to use are the first three: <code>ordered</code>, <code>maxPacketLifeTime</code>, and <code>maxRetransmits</code>. With <a href="https://en.wikipedia.org/wiki/Stream_Control_Transmission_Protocol" title="Wikipedia article about SCTP">SCTP</a> (now used by all browsers that support WebRTC) reliable and ordered is true by default. It makes sense to use unreliable and unordered if you want full control from the app layer, but in most cases, partial reliability is helpful.</p>
 
-<p>Note that, as with WebSocket, RTCDataChannel fires events when a connection is established, closed, or errors, and when it receives a message from the other peer.</p>
+<p>Note that, as with WebSocket, <code>RTCDataChannel</code> fires events when a connection is established, closed, or errors, and when it receives a message from the other peer.</p>
 
 <h2 id="is-it-safe">Is it safe?</h2>
 
-<p>Encryption is mandatory for all WebRTC components. With RTCDataChannel all data is secured with <a href="https://en.wikipedia.org/wiki/Datagram_Transport_Layer_Security" title="Wikipedia article about DTLS">Datagram Transport Layer Security</a> (DTLS). DTLS is a derivative of SSL, meaning your data will be as secure as using any standard SSL based connection. DTLS is standardized and built in to all browsers that support WebRTC. You can find more information about DTLS on the <a href="http://wiki.wireshark.org/DTLS">Wireshark wiki</a>.</p>
+<p>Encryption is mandatory for all WebRTC components. With <code>RTCDataChannel</code>, all data is secured with <a href="https://en.wikipedia.org/wiki/Datagram_Transport_Layer_Security" title="Wikipedia article about DTLS">Datagram Transport Layer Security (DTLS)</a>. DTLS is a derivative of SSL, meaning your data will be as secure as using any standard SSL-based connection. DTLS is standardized and built into all browsers that support WebRTC. For more information, see <a href="https://wiki.wireshark.org/DTLS">DTLS - The Wireshark wiki</a>.</p>
```

Comment:
I think DTLS should probably be left out of the link text, since it's not in the article title.

Reply:
Similar for the Wireshark link:

```
For more information, see the <a href="https://gitlab.com/wireshark/wireshark/-/wikis/DTLS">Wireshark Wiki DTLS article</a>.
```

Reply:
Will fix